interface/src/index.html

```diff
@@ -74,19 +74,22 @@
                 Download
               </a>
             </li>
-            <li ui-sref-active="active">
+            <li
+            ng-class="{active: inStateArr(['analyze', 'experiment'])}">
```

Comment:
This is super minor, but I notice that the `ng-class` here (and in all of the changes in this file) is lined up with the `<` in the line above, where as in `analysis/sampleBin.tpl.html`, it is offset by one space, so it is lined up with the `li` (which I think makes slightly more sense). What do you think? As I said, it is super minor, so I don't have really strong feelings about it.

Reply:
Nice catch. I will make it consistent.

- [ ] fix indentation at `ng-class` in index.html